freeorion.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>freeorion.desktop</id>
+  <name>FreeOrion</name>
+  <summary>Turn-based space empire and galactic conquest (4X) computer game</summary>
+  <summary xml:lang="de">Rundenbasierendes Weltraum-Strategiespiel</summary>
+  <developer_name>FreeOrion Project</developer_name>
+  <description>
+      <p>
+       FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game.
+      </p>
+  </description>
+  <keywords>
+    <keyword>game</keyword>
+    <keyword>strategy</keyword>
```

Comment:
There is not additional category `strategy` but `StrategyGame`; see https://specifications.freedesktop.org/menu-spec/latest/apas02.html

Also crosscheck with `freeorion.desktop` required.

Reply:
`freeorion.desktop` is already correct.

Reply:
Fixed 4675bd22738d693b9c59cb3f79a8b5f768232d7f

Reply:
Compare: https://github.com/freeorion/freeorion/blob/master/freeorion.desktop#L14 This may be redundant here.

Reply:
The idea here is that apps can specify software-center-only categories or keywords in the appdata, and the ones shared with users shell get put in the existing desktop file. Most apps don't need extra categories or keywords.